CONTRIBUTING.md

```diff
@@ -61,31 +61,91 @@ You can also clean up generated files:
-    MINOR version when you add functionality in a backwards compatible manner, and
-    PATCH version when you make backwards compatible bug fixes.
+To release a new package version, create a pull request which **modifies ONLY the package you wish to release**, and satisfies **ONE** of the following:
+
+- If the branch contains multiple commits, the pull request must have a [conventional title](#conventional-commit)
+- If the branch contains only a single commit...
+  - ... the commit itself must have a [conventional commit message](#conventional-commit), and...
+  - ... (for consistency) the PR title must match that commit message
+
+Once that pull request is merged into master, an automated release will be triggered, and the newly published package version will be available on the npm public registry.
+
+### Semantic Versioning
+
+All npm packages follow strict semantic versioning (or _semver_). Semantic versioning gives specific meaning to changes in version numbers, as follows:
+
+> Given a version number `MAJOR.MINOR.PATCH`, increment the:
+>
+> - `MAJOR` version when you make incompatible API changes,
+> - `MINOR` version when you add functionality in a backwards compatible manner, and
+> - `PATCH` version when you make backwards compatible bug fixes.
 
 — <https://semver.org/>
 
-If you make any breaking changes, include "BREAKING CHANGE" in the commit message. For example: **BREAKING CHANGE: removed feature from component X**.
+**Note that we do not update package version numbers directly**, but instead depend on a [Conventional Commit](#conventional-commit) workflow which will version and release packages according to the content of our pull requests.
+
+### Conventional Commit
+
+In order to provide versioning information to our automated release workflow, we ensure that our pull requests are structured in a way that the CI pipeline is able to infer the required version update (as well as which packages are to be released with that update) from the content of our pull requests.
+
+We enforce constraints ensuring that all pull requests will result in a merge commit to the master branch which satisfies Conventional Commit (according to the [Conventional Commit 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/) spec), by ensuring that either the pull request's title and/or commit messages satisfy the conditions of the [release workflow outlined above](#release-workflow).
+
+These constraints require that each pull request have a title formatted as follows:
+
+```
+<type>: <description>
+```
+
+#### Description
+
+The `description` should be a short (less than ~60 characters) summary of changes introduced in the release. This summary will be included in the CHANGELOG (see the [releases page](https://github.com/cultureamp/kaizen-design-system/releases)), along with the commit type, a link to the pull request which triggered the release, and a detailed summary of any breaking changes.
```

Comment:
This is a good inclusion, explaining the implication of how the description is used in the CHANGELOG.